src/App.js

```diff
@@ -1,7 +1,7 @@
 import Screen from './components/Screen';
 import Wrapper from './components/wrapper';
 import ButtonBox from './components/ButtonBox';
-import Button from './components/Button';
+import Button from './components/Calculator';
```

Comment:
[OPTIONAL] Kindly use the professional version of the Readme file. Other than that your project is complete!